environment/src/test/java/com/sequenceiq/environment/network/service/SubnetIdProviderTest.java

```diff
@@ -53,9 +54,9 @@ void testProvideThenNetworkSelectorCalled() {
         NetworkConnector networkConnector = setupConnectorWithSelectionResult(List.of(new CloudSubnet("id", "name")));
         Tunnel tunnel = Tunnel.DIRECT;
 
-        String provide = underTest.provide(networkDto, tunnel, CloudPlatform.AWS);
+        ProvidedSubnetIds provide = underTest.provide(networkDto, tunnel, CloudPlatform.AWS, false);
```

Comment:
In case you decide to rename `SubnetIdProvider.provide()` as per earlier suggestions, it might make sense to rename the related variables & test method names here as well.